.rubocop.yml

```diff
@@ -1,7 +1,13 @@
 AllCops:
-  TargetRubyVersion: 2.1
```

Comment:
🙏 https://github.com/k1LoW/awsecrets/pull/24#issuecomment-568826643

Reply:
I wasn't able to build this change with Ruby 2.1 (on Travis) :(